### PR TITLE
Replace whitelist/blacklist terminology with allowlist/denylist

### PR DESCRIPTION
**Pull request by peck**

### CODE REVIEW

File: lib/logger/lib/logger.ex [hunk 1]
```diff
@@ -109,7 +109,7 @@ defmodule Logger do
 
       Logger.error("We have a problem", [error_code: :pc_load_letter])
 
-  In your app's logger configuration, you would need to whitelist the
+  In your app's logger configuration, you would need to allowlist the
```
Comment (josevalim) [inline lib/logger/lib/logger.ex:112]:
```suggestion
  In your app's logger configuration, you would need to include the
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by MoacirSchmidt** [comment]
this is unbelievable ridiculous. So many to do in computer science and some people leftist biased guide us to a waste o time

**Comment by gusaiani** [comment]
For the record, I think this was a worthy initiative.
I am certainly not a leftist.

**Pull request merged**